accounts/backends.py

```diff
@@ -59,8 +59,8 @@ def authenticate(self, request, remote_user, tokens=True):
                 user.is_staff = False
                 user.is_superuser = False
 
-        # Update groups: first remove them from existing groups then add them to new groups
-        user.groups.filter(name__startswith="platform_").delete()
+        # First disassociates user with platform groups, then loads in new groups
+        user.groups.remove(*user.groups.filter(name__startswith="platform_"))
```

Comment:
To be honest, I have no clue what this code does but I'm gonna assume it has been tested and works 😆 👍